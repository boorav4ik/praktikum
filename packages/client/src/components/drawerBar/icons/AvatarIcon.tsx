import { SvgIcon, SvgIconProps } from '@mui/material'

interface AvatarIconProps extends SvgIconProps {
  open?: boolean
}

export default function AvatarIcon(props: AvatarIconProps) {
  const viewBox = props.open ? '-0 0 70 70' : '-10 -10 90 90'
  return (
    <SvgIcon {...props} viewBox={viewBox}>
      <path
        d="M35 7C19.5544 7 7 19.5965 7 35.1761C7 39.5943 8.00806 43.7651 9.80151 47.4775C10.6424 49.218 9.91303 51.3107 8.1725 52.1515C6.43197 52.9924 4.33934 52.263 3.49849 50.5225C1.25579 45.8802 0 40.6704 0 35.1761C0 15.7673 15.6517 0 35 0C54.3483 0 70 15.7673 70 35.1761C70 40.6704 68.7442 45.8802 66.5015 50.5225C65.6607 52.263 63.568 52.9924 61.8275 52.1515C60.087 51.3107 59.3576 49.218 60.1985 47.4775C61.9919 43.7651 63 39.5943 63 35.1761C63 19.5965 50.4456 7 35 7Z"
        fill="#1E515D"
      />
      <path
        fill-rule="evenodd"
        clip-rule="evenodd"
        d="M43.7902 50.1355C48.9988 47.1039 52.5 41.4608 52.5 35C52.5 25.335 44.665 17.5 35 17.5C25.335 17.5 17.5 25.335 17.5 35C17.5 41.4608 21.0012 47.1039 26.2098 50.1355C20.8525 51.5872 16.754 54.2324 13.7813 56.8912C11.5839 58.8566 10.0055 60.8229 8.96817 62.3116C8.44821 63.0579 8.0601 63.6901 7.79464 64.1506C7.66178 64.3811 7.55928 64.5693 7.48599 64.708C7.44934 64.7774 7.41996 64.8345 7.39772 64.8784L7.36968 64.9343L7.35963 64.9546L7.35559 64.9629L7.35382 64.9665C7.353 64.9682 7.35221 64.9698 10.5 66.5L7.35221 64.9698C6.50712 66.7083 7.23135 68.8027 8.96983 69.6478C10.7036 70.4906 12.7913 69.7726 13.6409 68.0443L13.6429 68.0403L13.6755 67.9776C13.7106 67.9112 13.7715 67.7986 13.8593 67.6463C14.0352 67.3412 14.317 66.8796 14.7115 66.3134C15.5032 65.1771 16.7321 63.6434 18.4479 62.1088C21.846 59.0695 27.1697 56 35 56C42.8303 56 48.154 59.0695 51.5521 62.1088C53.2679 63.6434 54.4968 65.1771 55.2885 66.3134C55.683 66.8796 55.9648 67.3412 56.1407 67.6463C56.2285 67.7986 56.2894 67.9112 56.3245 67.9776L56.3571 68.0403L56.3591 68.0442C57.2086 69.7726 59.2964 70.4906 61.0302 69.6478C62.7686 68.8027 63.4929 66.7083 62.6478 64.9698L59.5 66.5C62.6478 64.9698 62.647 64.9682 62.6462 64.9665L62.6404 64.9546L62.6303 64.9343L62.6023 64.8784C62.58 64.8345 62.5507 64.7774 62.514 64.708C62.4407 64.5693 62.3382 64.3811 62.2054 64.1506C61.9399 63.6901 61.5518 63.0579 61.0318 62.3116C59.9945 60.8229 58.4161 58.8566 56.2187 56.8912C53.246 54.2324 49.1475 51.5872 43.7902 50.1355ZM35 45.5C40.799 45.5 45.5 40.799 45.5 35C45.5 29.201 40.799 24.5 35 24.5C29.201 24.5 24.5 29.201 24.5 35C24.5 40.799 29.201 45.5 35 45.5Z"
        fill="#1E515D"
      />
    </SvgIcon>
  )
}
