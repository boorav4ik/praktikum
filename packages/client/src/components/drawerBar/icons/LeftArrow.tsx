import { SvgIcon, SvgIconProps } from '@mui/material'

const D = 'M17.77 3.77 16 2 6 12l10 10 1.77-1.77L9.54 12z'

export default function LeftArrow(props: SvgIconProps) {
  return (
    <SvgIcon {...props} viewBox="0 0 22 22">
      <path d={D} />
    </SvgIcon>
  )
}
