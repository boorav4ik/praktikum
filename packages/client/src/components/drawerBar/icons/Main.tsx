import { SvgIcon, SvgIconProps } from '@mui/material'

const D = 'M10 20v-6h4v6h5v-8h3L12 3 2 12h3v8z'

export default function Main(props: SvgIconProps) {
  return (
    <SvgIcon {...props} viewBox="0 0 22 22">
      <path d={D} />
    </SvgIcon>
  )
}
