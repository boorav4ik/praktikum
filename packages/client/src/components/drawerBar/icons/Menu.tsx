import { SvgIcon, SvgIconProps } from '@mui/material'

const D = 'M3 18h18v-2H3v2zm0-5h18v-2H3v2zm0-7v2h18V6H3z'

export default function MenuIcon(props: SvgIconProps) {
  return (
    <SvgIcon {...props} viewBox="0 0 22 22">
      <path d={D} />
    </SvgIcon>
  )
}
