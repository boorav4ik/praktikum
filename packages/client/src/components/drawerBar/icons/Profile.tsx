import { SvgIcon, SvgIconProps } from '@mui/material'

const D =
  'M12 12c2.21 0 4-1.79 4-4s-1.79-4-4-4-4 1.79-4 4 1.79 4 4 4zm0 2c-2.67 0-8 1.34-8 4v2h16v-2c0-2.66-5.33-4-8-4z'

export default function ProfileIcon(props: SvgIconProps) {
  return (
    <SvgIcon {...props} viewBox="0 0 22 22">
      <path d={D} />
    </SvgIcon>
  )
}
