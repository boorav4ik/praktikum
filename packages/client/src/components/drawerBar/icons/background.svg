<svg width="1440" height="100" viewBox="0 0 1440 100" fill="none" xmlns="http://www.w3.org/2000/svg">
<rect x="181.463" y="-40.4595" width="115.347" height="115.347" rx="5" transform="rotate(15 181.463 -40.4595)" fill="#7AB3A2"/>
<rect x="82.9127" y="-8.71985" width="94.3433" height="94.3433" rx="5" transform="rotate(-38.2994 82.9127 -8.71985)" fill="#7AB3A2"/>
<rect x="-39.8437" y="-16" width="70" height="70" rx="5" transform="rotate(6.69125 -39.8437 -16)" fill="#7AB3A2"/>
<rect x="233.156" y="-22" width="70" height="70" rx="5" transform="rotate(6.69125 233.156 -22)" fill="#7AB3A2"/>
<rect x="836.183" y="-12.9857" width="98.3328" height="98.3328" rx="5" transform="rotate(6.69125 836.183 -12.9857)" fill="#7AB3A2"/>
<rect x="1243.16" y="-12" width="70" height="70" rx="5" transform="rotate(6.69125 1243.16 -12)" fill="#7AB3A2"/>
<rect x="-2.83127" y="10.6757" width="100.367" height="100.367" rx="5" transform="rotate(-25.795 -2.83127 10.6757)" fill="#7AB3A2"/>
<rect x="273.577" y="-12.8971" width="87.0577" height="87.0577" rx="5" transform="rotate(-25.795 273.577 -12.8971)" fill="#7AB3A2"/>
<rect x="357.324" y="-31.938" width="108.068" height="108.068" rx="5" transform="rotate(14.0274 357.324 -31.938)" fill="#7AB3A2"/>
<rect x="431.677" y="5.66528" width="97.405" height="97.405" rx="5" transform="rotate(-13.4554 431.677 5.66528)" fill="#7AB3A2"/>
<rect x="518.964" y="-0.0124512" width="90.901" height="90.901" rx="5" transform="rotate(-26.8299 518.964 -0.0124512)" fill="#7AB3A2"/>
<rect x="613.442" y="-15.5944" width="100.769" height="100.769" rx="5" transform="rotate(9.46728 613.442 -15.5944)" fill="#7AB3A2"/>
<rect x="676.659" y="-3.92896" width="93.9411" height="93.9411" rx="5" transform="rotate(-15 676.659 -3.92896)" fill="#7AB3A2"/>
<rect x="778.567" y="-21.7799" width="88.7619" height="98.2617" rx="5" transform="rotate(19.5211 778.567 -21.7799)" fill="#7AB3A2"/>
<rect x="814.229" y="-19.5435" width="96.3915" height="96.3915" rx="5" transform="rotate(1.05378 814.229 -19.5435)" fill="#7AB3A2"/>
<rect x="907.992" y="-11.33" width="114.081" height="114.081" rx="5" transform="rotate(-8.40221 907.992 -11.33)" fill="#7AB3A2"/>
<rect x="1019.55" y="-9" width="84.0745" height="84.0745" rx="5" transform="rotate(2.41797 1019.55 -9)" fill="#7AB3A2"/>
<rect x="1087.73" y="-0.375366" width="103.642" height="103.642" rx="5" transform="rotate(-10.3522 1087.73 -0.375366)" fill="#7AB3A2"/>
<rect x="1199.58" y="-31" width="106.572" height="106.572" rx="5" transform="rotate(15 1199.58 -31)" fill="#7AB3A2"/>
<rect x="1385.44" y="-32.6399" width="91.9624" height="91.9624" rx="5" transform="rotate(7.69234 1385.44 -32.6399)" fill="#7AB3A2"/>
<rect x="1291.23" y="4.65552" width="98.8698" height="98.8698" rx="5" transform="rotate(-10.8657 1291.23 4.65552)" fill="#7AB3A2"/>
</svg>
