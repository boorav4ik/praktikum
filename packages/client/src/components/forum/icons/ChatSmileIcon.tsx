import { SvgIcon, SvgIconProps } from '@mui/material'

const d =
  'M0.367077 24.6268C0.0337002 24.295 -0.0841095 23.8041 0.0612728 23.357L2.07407 17.1255C-0.0803492 12.5407 0.849596 7.13696 4.46661 3.52277C6.74009 1.25107 9.76303 0 12.9777 0C16.1924 0 19.2154 1.25107 21.4876 3.52277C22.4577 4.49206 23.2548 5.63042 23.8563 6.90403C24.1509 7.53019 23.8827 8.27532 23.2573 8.57086C22.6319 8.86641 21.8824 8.59716 21.5891 7.97226C21.1116 6.96038 20.4812 6.05997 19.7155 5.2948C17.9157 3.49647 15.5219 2.50589 12.9777 2.50589C10.4323 2.50589 8.03975 3.49647 6.23877 5.2948C3.27598 8.25653 2.59042 12.7373 4.53429 16.4454C4.69095 16.7435 4.71977 17.0916 4.61575 17.411L3.19827 21.7978L7.58731 20.3815C7.90941 20.28 8.25532 20.3088 8.5536 20.4641C12.2634 22.4065 16.7502 21.7227 19.7142 18.761C21.6092 16.8675 22.593 14.3415 22.4865 11.6453C22.4589 10.954 22.9966 10.3717 23.6884 10.3441C24.3915 10.3354 24.9643 10.8538 24.9906 11.5451C25.1234 14.8976 23.8463 18.1724 21.4864 20.5305C17.8706 24.1447 12.4639 25.0764 7.87181 22.9212L1.63917 24.9324C0.840823 25.1904 0.367077 24.6268 0.367077 24.6268ZM18.5311 8.73993C18.5311 8.04865 17.9709 7.48761 17.2778 7.48761H15.0971C14.404 7.48761 13.8438 8.04865 13.8438 8.73993C13.8438 9.43121 14.404 9.99225 15.0971 9.99225H17.2778C17.9696 9.99225 18.5311 9.43121 18.5311 8.73993ZM12.2646 8.73993C12.2646 8.04865 11.7044 7.48761 11.0113 7.48761H8.83058C8.13751 7.48761 7.57729 8.04865 7.57729 8.73993C7.57729 9.43121 8.13751 9.99225 8.83058 9.99225H11.0113C11.7031 9.99225 12.2646 9.43121 12.2646 8.73993ZM19.8784 13.5288C20.03 12.8538 19.6064 12.1838 18.9309 12.031C18.2666 11.8782 17.5861 12.3028 17.4332 12.9765C16.9557 15.0904 15.1108 16.5657 12.9477 16.5657C10.7845 16.5657 8.93962 15.0892 8.46211 12.9765C8.30921 12.3003 7.62993 11.8782 6.96443 12.031C6.28765 12.1838 5.86403 12.855 6.01568 13.5288C6.75262 16.7923 9.60261 19.0716 12.9464 19.0716C16.2902 19.0716 19.1414 16.7923 19.8784 13.5288Z'
export type SvgIconUserProp = SvgIconProps & { customColor?: string }

export function ChatSmileIcon(props?: SvgIconUserProp) {
  return (
    <SvgIcon viewBox="0 0 25 25" {...props}>
      <path d={d} fill={props?.customColor ?? 'white'} />
    </SvgIcon>
  )
}
