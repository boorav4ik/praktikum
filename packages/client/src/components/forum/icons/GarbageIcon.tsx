import { SvgIcon, SvgIconProps } from '@mui/material'

const d =
  'M341.333,77.576V0H170.667v77.576H23.273v46.545h38.788V512h387.879V124.121h38.788V77.576H341.333z M217.212,46.545 h77.576v31.03h-77.576V46.545z M403.394,465.455H108.606V124.121h294.788V465.455z'
export type SvgIconUserProp = SvgIconProps & { customColor?: string }

export default function GarbageIcon(props?: SvgIconUserProp) {
  return (
    <SvgIcon {...props}>
      <path d={d} fill={props?.customColor ?? 'white'} />
    </SvgIcon>
  )
}
