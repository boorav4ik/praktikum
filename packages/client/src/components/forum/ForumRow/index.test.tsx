import { render, screen, fireEvent } from '@testing-library/react'
import {ForumRow} from './index'
import {ChatAnswerIcon} from '../icons/ChatAnswerIcon'

const text = 'test'
const btnText = 'testBtn'
const onClick = jest.fn()
describe('ForumRow component', () => {
  it('renders', () => {
    render(<ForumRow text={text} btnText={btnText} onClick={onClick}/>)

    expect(screen.getByText(text)).toBeInTheDocument()
    expect(screen.getByText(btnText)).toBeInTheDocument()
  })

  it('calls onClick', () => {
    const { getByRole } = render(<ForumRow text={text} btnText={btnText} onClick={onClick}/>)
    const btn = getByRole('button')

    fireEvent.click(btn)
    expect(onClick).toHaveBeenCalledTimes(1)
  })

  it('renders ChatAnswerIcon', () => {
    render(<ForumRow text={text} btnText={btnText} onClick={onClick} icon={ChatAnswerIcon}/>)

    expect(screen.getByRole('path')).toBeInTheDocument()
  })
})
export {}