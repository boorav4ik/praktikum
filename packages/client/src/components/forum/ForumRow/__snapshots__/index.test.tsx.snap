// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ForumRow component ForumRow snapshot 1`] = `
Object {
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <div
        class="MuiGrid-root MuiGrid-container MuiGrid-item css-ejktsk-MuiGrid-root"
      >
        <div
          class="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-10 css-17p3wh3-MuiGrid-root"
        >
          <div
            class="MuiBox-root css-12iez47"
          >
            test
          </div>
        </div>
        <div
          class="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-2 css-1o7apob-MuiGrid-root"
        >
          <button
            class="MuiButtonBase-root MuiButton-root MuiButton-contained MuiButton-containedPrimary MuiButton-sizeMedium MuiButton-containedSizeMedium MuiButton-disableElevation MuiButton-root MuiButton-contained MuiButton-containedPrimary MuiButton-sizeMedium MuiButton-containedSizeMedium MuiButton-disableElevation css-1sx6bci-MuiButtonBase-root-MuiButton-root"
            tabindex="0"
            type="button"
          >
            <svg
              aria-hidden="true"
              class="MuiSvgIcon-root MuiSvgIcon-fontSizeMedium css-i4bv87-MuiSvgIcon-root"
              focusable="false"
              viewBox="0 0 25 25"
            >
              <path
                d="M6.14976 4.27879C6.68984 3.84715 6.75985 3.05267 6.32229 2.51843C5.88347 1.98294 5.09585 1.90662 4.56327 2.34577C4.56327 2.34577 1.88912 4.26253 0.98398 8.24617C0.98398 8.24617 0.128851 10.9874 1.72659 14.6871L0.0600902 19.8793C-0.0836815 20.326 0.0350865 20.8164 0.367637 21.148C0.367637 21.148 0.901467 21.6922 1.63658 21.452L6.45856 19.8831C6.82236 20.4348 7.22117 20.964 7.69374 21.437C10.5867 24.3309 14.8998 25.0928 18.5779 23.4113L23.3674 24.9589C23.3674 24.9589 24.1512 25.1654 24.6351 24.6524C24.9576 24.3109 25.0851 23.8304 24.9414 23.3825L23.3949 18.5906C23.3949 18.5906 25.1001 15.5178 23.7524 11.4266C23.7524 11.4266 23.0698 9.14449 21.0233 7.33909C20.5344 5.72261 19.653 4.24126 18.4229 3.01639C15.935 0.535368 12.3907 -0.508086 8.94393 0.235094C8.26883 0.380226 7.83877 1.04458 7.98379 1.72145C8.12881 2.39582 8.79141 2.83122 9.46901 2.68233C12.0844 2.11806 14.7723 2.91004 16.6588 4.79301C18.1441 6.27186 18.9629 8.24116 18.9667 10.3381C18.9704 12.4362 18.1578 14.4093 16.6776 15.8944C14.241 18.3416 10.5504 18.9109 7.49246 17.312C7.19492 17.1568 6.84737 17.1293 6.52732 17.2319L3.18681 18.3166L4.26073 14.9711C4.36324 14.6508 4.32574 14.308 4.17696 14.0064C4.17696 14.0064 2.81176 11.389 3.43685 8.73036C3.43685 8.72911 3.85816 6.11046 6.14976 4.27879ZM18.8667 20.8765C18.5491 20.7701 18.2016 20.8014 17.9028 20.9578C15.5949 22.1701 12.9008 22.0125 10.7655 20.6763C13.5709 20.7776 16.3613 19.7554 18.4491 17.6585C19.9793 16.1209 20.9595 14.1866 21.3133 12.0884C22.0009 13.9539 21.9146 16.0596 20.9432 17.9138C20.787 18.2103 20.7582 18.5593 20.8607 18.8784L21.8121 21.8248L18.8667 20.8765ZM16.135 8.59899C16.135 9.29087 15.5749 9.85013 14.8848 9.85013H7.15241C6.46231 9.85013 5.90222 9.29087 5.90222 8.59899C5.90222 7.90711 6.46231 7.34785 7.15241 7.34785H14.8848C15.5749 7.34785 16.135 7.90711 16.135 8.59899ZM16.135 12.3699C16.135 13.0618 15.5749 13.6211 14.8848 13.6211H7.15241C6.46231 13.6211 5.90222 13.0618 5.90222 12.3699C5.90222 11.6781 6.46231 11.1188 7.15241 11.1188H14.8848C15.5749 11.1188 16.135 11.6781 16.135 12.3699Z"
                fill="white"
              />
            </svg>
            <span
              style="margin-left: 5px;"
            >
              testBtn
            </span>
            <span
              class="MuiTouchRipple-root css-8je8zh-MuiTouchRipple-root"
            />
          </button>
        </div>
      </div>
    </div>
  </body>,
  "container": <div>
    <div
      class="MuiGrid-root MuiGrid-container MuiGrid-item css-ejktsk-MuiGrid-root"
    >
      <div
        class="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-10 css-17p3wh3-MuiGrid-root"
      >
        <div
          class="MuiBox-root css-12iez47"
        >
          test
        </div>
      </div>
      <div
        class="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-2 css-1o7apob-MuiGrid-root"
      >
        <button
          class="MuiButtonBase-root MuiButton-root MuiButton-contained MuiButton-containedPrimary MuiButton-sizeMedium MuiButton-containedSizeMedium MuiButton-disableElevation MuiButton-root MuiButton-contained MuiButton-containedPrimary MuiButton-sizeMedium MuiButton-containedSizeMedium MuiButton-disableElevation css-1sx6bci-MuiButtonBase-root-MuiButton-root"
          tabindex="0"
          type="button"
        >
          <svg
            aria-hidden="true"
            class="MuiSvgIcon-root MuiSvgIcon-fontSizeMedium css-i4bv87-MuiSvgIcon-root"
            focusable="false"
            viewBox="0 0 25 25"
          >
            <path
              d="M6.14976 4.27879C6.68984 3.84715 6.75985 3.05267 6.32229 2.51843C5.88347 1.98294 5.09585 1.90662 4.56327 2.34577C4.56327 2.34577 1.88912 4.26253 0.98398 8.24617C0.98398 8.24617 0.128851 10.9874 1.72659 14.6871L0.0600902 19.8793C-0.0836815 20.326 0.0350865 20.8164 0.367637 21.148C0.367637 21.148 0.901467 21.6922 1.63658 21.452L6.45856 19.8831C6.82236 20.4348 7.22117 20.964 7.69374 21.437C10.5867 24.3309 14.8998 25.0928 18.5779 23.4113L23.3674 24.9589C23.3674 24.9589 24.1512 25.1654 24.6351 24.6524C24.9576 24.3109 25.0851 23.8304 24.9414 23.3825L23.3949 18.5906C23.3949 18.5906 25.1001 15.5178 23.7524 11.4266C23.7524 11.4266 23.0698 9.14449 21.0233 7.33909C20.5344 5.72261 19.653 4.24126 18.4229 3.01639C15.935 0.535368 12.3907 -0.508086 8.94393 0.235094C8.26883 0.380226 7.83877 1.04458 7.98379 1.72145C8.12881 2.39582 8.79141 2.83122 9.46901 2.68233C12.0844 2.11806 14.7723 2.91004 16.6588 4.79301C18.1441 6.27186 18.9629 8.24116 18.9667 10.3381C18.9704 12.4362 18.1578 14.4093 16.6776 15.8944C14.241 18.3416 10.5504 18.9109 7.49246 17.312C7.19492 17.1568 6.84737 17.1293 6.52732 17.2319L3.18681 18.3166L4.26073 14.9711C4.36324 14.6508 4.32574 14.308 4.17696 14.0064C4.17696 14.0064 2.81176 11.389 3.43685 8.73036C3.43685 8.72911 3.85816 6.11046 6.14976 4.27879ZM18.8667 20.8765C18.5491 20.7701 18.2016 20.8014 17.9028 20.9578C15.5949 22.1701 12.9008 22.0125 10.7655 20.6763C13.5709 20.7776 16.3613 19.7554 18.4491 17.6585C19.9793 16.1209 20.9595 14.1866 21.3133 12.0884C22.0009 13.9539 21.9146 16.0596 20.9432 17.9138C20.787 18.2103 20.7582 18.5593 20.8607 18.8784L21.8121 21.8248L18.8667 20.8765ZM16.135 8.59899C16.135 9.29087 15.5749 9.85013 14.8848 9.85013H7.15241C6.46231 9.85013 5.90222 9.29087 5.90222 8.59899C5.90222 7.90711 6.46231 7.34785 7.15241 7.34785H14.8848C15.5749 7.34785 16.135 7.90711 16.135 8.59899ZM16.135 12.3699C16.135 13.0618 15.5749 13.6211 14.8848 13.6211H7.15241C6.46231 13.6211 5.90222 13.0618 5.90222 12.3699C5.90222 11.6781 6.46231 11.1188 7.15241 11.1188H14.8848C15.5749 11.1188 16.135 11.6781 16.135 12.3699Z"
              fill="white"
            />
          </svg>
          <span
            style="margin-left: 5px;"
          >
            testBtn
          </span>
          <span
            class="MuiTouchRipple-root css-8je8zh-MuiTouchRipple-root"
          />
        </button>
      </div>
    </div>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;
